learn/quickstart/hello_world.md

```diff
@@ -22,7 +22,7 @@ which is part of the
 
 To install gfortran on Linux, use your system package manager.
 Otherwise, for macOS or Windows, refer to gfortran binaries from 
-[this page](https://gcc.gnu.org/install/binaries.html).
+[this page](http://www.equation.com/servlet/equation.cmd?fa=fortran).
```

Comment:
The equation.com links seems to serve only Windows binaries. This suggestion adds resources for macOS.

```suggestion
On macOS, you can install gfortran using [Homebrew](https://brew.sh/) or [MacPorts](https://www.macports.org/).
On Windows, you can get native binaries [here](http://www.equation.com/servlet/equation.cmd?fa=fortran).
```

Reply:
@milancurcic,

Thank you for including the macOS in this tutorial. Do we need someone to test this before committing? I do not have a means of testing the macOS setup.

Thanks.